internal/util/util.go

```diff
@@ -337,3 +346,75 @@ func contains(s []string, key string) bool {
 
 	return false
 }
+
+// DefaultLog helps in logging with klog.level 1
+func DefaultLog(message string, args ...interface{}) {
+	logMessage := fmt.Sprintf(message, args...)
+	// If logging is disabled, don't evaluate the arguments
+	if klog.V(Default).Enabled() {
+		klog.InfoDepth(1, logMessage)
+	}
+}
+
+// UsefulLog helps in logging with klog.level 2
+func UsefulLog(ctx context.Context, message string, args ...interface{}) {
+	logMessage := fmt.Sprintf(Log(ctx, message), args...)
+	// If logging is disabled, don't evaluate the arguments
+	if klog.V(Useful).Enabled() {
+		klog.InfoDepth(1, logMessage)
+	}
+}
+
+// ExtendedLogMsg helps in logging a message with klog.level 3
+func ExtendedLogMsg(message string, args ...interface{}) {
+	logMessage := fmt.Sprintf(message, args...)
+	// If logging is disabled, don't evaluate the arguments
+	if klog.V(Default).Enabled() {
```

Comment:
do we need to check this one in all cases?, Default is checked here

Reply:
Oh I missed that, I will update.

Reply:
Its there in other places also